### ISSUE TITLE
Use the "What's new in Android Studio" panel for release notes

### ISSUE THREAD

**Issue by InMatrix** [flutter/flutter-intellij#2708]
When I upgraded Android Studio to 3.2, it showed me this nice graphical release notes in a side panel (screenshot below). Can we use this to provide better release notes after a new version of the Flutter plugin is installed? The panel can be re-opened via `Help > What's New in Android Studio`.

![image](https://user-images.githubusercontent.com/348942/47233928-0274f600-d389-11e8-939a-74cbe8609880.png)

**Comment by pq** [comment]
That would be great.  I'm not sure if that's something we can contribute to...

@stevemessick : any idea?

**Comment by stevemessick** [comment]
Not off hand; I'll have to see if they defined a contribution mechanism. We wouldn't want to overwrite it but if we can add to it that might be nice. Or maybe we want a `Help > What's New in Flutter` as a separate panel.

@devoncarew Seems like it would really help discovery of new stuff.

**Comment by InMatrix** [comment]
> Or maybe we want a Help > What's New in Flutter as a separate panel.
That works, too. It's probably cleaner than adding to "What's New in Android Studio".

**Comment by devoncarew** [comment]
Yup, I agree that this would makes sense as a new `What's New in Flutter` panel.

**Comment by stevemessick** [comment]
The Assistant framework can be used for more than a "What's New" panel. Go to Tools > Firebase for a much more elaborate example. It basically implements a built-in tutorial that does some amount of code editing for you, assuming you have an Android (not Flutter) project open.

As far as making a "What's New" for Flutter, it looks straight-forward. We'd need to implement some new classes and define the content to display in an XML file.

**Comment by InMatrix** [comment]
Let's give it a try for the next plugin release?

Cc: @mjohnsullivan We can use some help from DevRel to make this space more useful.

**Comment by mjohnsullivan** [comment]
Certainly; we can help.

**Comment by InMatrix** [comment]
Cc: @csells
